docs/rules/no_console_log.md

```diff
@@ -0,0 +1,25 @@
+# no_console_log
+
+Disallows the use of `console.log`.
+
+Oftentimes, developers are guilty of committing `console.log` statements accidentally, left in particularly after debugging. Moreover, using `console.log` in code may leak sensitive information to the output or clutter the console with unnecessary information. This rule helps maintain clean and secure code by disallowing the use of `console.log`.
```

Comment:
I feel like putting a negative word like "guilty" is a bit much for official docs, given that this is usually done unintentionally. Maybe we an say something like: "Oftentimes, developers accidentally commit `console.log` statements, ..."